ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/apps/hierarchies/OntologyHierarchiesDisplayStrings.properties

```diff
@@ -0,0 +1 @@
+hierarchyLabelName = {0}
```

Comment:
Since we're going with just the label instead of "Apps by <label>", we can probably can get rid of the extra boiler plate.